.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.7.0
+2.7.0
```

Comment:
I'm curious what triggered this change?

Reply:
@rodri0315 and I think we've fixed this 👍